paper/paper.md

```diff
@@ -38,12 +38,12 @@ bibliography: paper.bib
 
 # Summary
 PyMilo is an open-source Python package that addresses the limitations of existing machine learning (ML) model storage formats by providing a transparent, reliable, end-to-end, and safe method for exporting and deploying trained models. 
-Current tools rely on black-box or executable formats that obscure internal model structures, making them difficult to audit, verify, or safely share. 
+Current tools rely on black-box or executable formats that obscure internal model structures, making them difficult to audit, verify, or safely share. Meanwhile, tensor-centric formats such as SafeTensors [@huggingface_safetensors_2022] securely store and transfer numerical tensors but do not capture the symbolic or structural composition of classical machine-learning models (e.g., scikit-learn pipelines), which remain PyMilo’s primary focus. 
```

Comment:
`symbolic` doesn't seem like a correct word here.
I'd say either remove them or change them to something like (`internal`)

Reply:
Good point